ResearchUXFactory/NSDate+ISO8601Format.m

```diff
@@ -72,6 +72,34 @@ + (NSDateFormatter *)ISO8601DateOnlyformatter
     return formatter;
 }
 
++ (NSDateFormatter *)UnitedStatesDateOnlyformatter
```

Comment:
Not sure I understand. Researchers are adding dates to the study? And the Web UI is converting them to US-specific date strings and not ISO8601 date-only strings?